projects/MTHN/index.js

```diff
@@ -0,0 +1,6 @@
+const { uniTvlExport } = require('../helper/unknownTokens')
+
+module.exports = uniTvlExport('bsc', '0xcC0D48a5530Cca0481105cCD61A14C495A51c901', {

```

Comment:
this is erc20 token address, not a dex

Reply:
Its an BEP 20 Token listed on Dex (Pancake Swap)
https://pancakeswap.finance/swap?inputCurrency=0xcC0D48a5530Cca0481105cCD61A14C495A51c901

https://coinmarketcap.com/currencies/mth-network/

Reply:
yes, but we dont list tokens but defi projects